ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -14,6 +14,12 @@ class BadgeLabelButton: UIButton {
         }
     }
 
+    var shouldUseWindowColorInBadge: Bool? {
```

Comment:
Now, I made it non-optional but just wanted to know that what is the problem to make this var optional?

If the intention is to make the caller forcibly set this value then should I go ahead and change the initializer itself? But again do we really need to do that?

Reply:
The rationale behind that is that we're already providing a default boolean value for that, so if the calling code is changing the value, it needs to be a valid boolean (instead of a possible nil value).